README.md

```diff
@@ -1,8 +1,27 @@
 # Deploy a Testnet
 
-## Software Setup \(Manual Installation\)
+## Setup
 
-- Install [GNU Wget](https://www.gnu.org/software/wget/):
+If you're running a full node validator we recommend you to comply with the necessary [technical requirements](https://github.com/cosmos/cosmos/blob/master/VALIDATORS_FAQ.md#technical-requirements). You can check more information regarding validators in our [website](https://cosmos.network/validators) or in the [validator FAQ](https://cosmos.network/resources/validator-faq).
```

Comment:
These requirements don't apply just yet. For the testnets you don't need to be in a datacenter. I think it would scare most people away.